Tasks/AzureCLIV1/azureclitask.ts

```diff
@@ -140,14 +141,8 @@ export class azureclitask {
     }
 
     private static setConfigDirectory(): void {
-        var configDirName: string = "c" + new Date().getTime(); // 'c' denotes config
-        if (tl.osType().match(/^Win/)) {
-            this.azCliConfigPath = path.join(process.env.USERPROFILE, ".azclitask", configDirName);
-        }
-        else {
-            this.azCliConfigPath = path.join(process.env.HOME, ".azclitask", configDirName);
-        }
-
+        var configDirName: string = "config" + new Date().getTime();
```

Comment:
I would still keep it at "c" just to keep the path short and not fall in windows path length restriction issue. Now that we are using Agent temp directory, it can be arbitrarily long.

Reply:
@bishal-pdMSFT The longest part of these path changes would be the date time. I could get rid of it and just set AZURE_CONFIG_DIR to agent's temp dir.

Reply:
I think timestamp is still required as there could be multiple CLI tasks in one job and hence multiple unique config dirs need to be created.

Reply:
You put in a finally action to remove the config folder when task is finishing, so another Azure CLI task would not see a previous task's config. I agree that we still need another folder, so it doesn't remove the entire temp dir.
```
if (this.azCliConfigPath) {
  tl.rmRF(this.azCliConfigPath);
}
```

Reply:
We can remove this logic of deleting config directory as anyways the agent's temp directory is cleaned up after build/release job.